core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts

```diff
@@ -158,5 +209,37 @@ describe('DotContainerPropertiesComponent', () => {
             expect(inplace.componentInstance.active).toBe(true);
             expect(title.attributes.dotAutofocus).toBeDefined();
         });
+
+        it('should setup title', () => {
+            const field = de.query(By.css('[data-testId="title"]'));
+            expect(field.attributes.pInputText).toBeDefined();
+        });
+
+        it('should setup description', () => {
+            const field = de.query(By.css('[data-testId="description"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should setup Max Contents', () => {
+            const field = de.query(By.css('[data-testId="max-contents"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should setup code', () => {
+            const field = de.query(By.css('[data-testId="code"]'));
+            expect(field).toBeDefined();
+        });
+
+        it('should button enable when max content greater then zero', fakeAsync(() => {
+            fixture.componentInstance.form.get('maxContentlets').setValue(2);
+            const contentTypeButton = de.query(By.css('[data-testId=showContentTypeAndCode]'));
+            contentTypeButton.nativeElement.click();
+            tick(3000);
```

Comment:
This is a clear sign this component is bad implemented, it should not take 3 seconds to show the component with the code and the tabs.

Reply:
It is not taking 3 seconds. I added this randomly removed this and update the test case.